src/components/core/posts/SinglePost.tsx

```diff
@@ -112,7 +126,7 @@ export default function SinglePost({
           <Image src="/assets/user.png" alt="user" width={40} height={40} />
         </div>
         <div>
-        <h5 className="font-medium text-md">{singlePost && singlePost?.accountId}</h5>
+        <h5 className="font-medium text-md">{user.name}</h5>
```

Comment:
I think it should be according to the user post and not according to the current user. One solution is that you are getting the `accountId` from the singlePost object, pass this id to the api which fetches the username/fullName